openmmtools/forces.py

```diff
@@ -256,8 +266,14 @@ class RadiallySymmetricRestraintForce(utils.RestorableOpenMMObject):
         The indices of the second group of atoms to restrain.
     controlling_parameter_name : str
         The name of the global parameter controlling the energy function.
+    energy_string : str
```

Comment:
Should we maybe call this `energy_function` for consistency with the other classes?